lib/model/Fixture.js

```diff
@@ -128,6 +128,32 @@ module.exports = class Fixture {
     return this._cache.allChannels;
   }
 
+  // returns object whose keys are all items in this.allChannelKeys
+  // { 'channel key': 'unique channel name' }
+  get uniqueChannelNames() {
+    if (!('uniqueChannelNames' in this._cache)) {
+      this._cache.uniqueChannelNames = {};
+
+      let names = this.allChannels.map(ch => ch.name || ch.key);
```

Comment:
Should we add a `name` getter in `SwitchingChannel` that also just returns the key to avoid this?